blog/zh/blog/2023/01/11/apisix-amesh-introduction.md

```diff
@@ -0,0 +1,87 @@
+---
+title: "基于 APISIX 的服务网格方案 Amesh 积极开发中！"
+authors:
+  - name: "lingsamuel"
+    title: "Author"
+    url: "https://github.com/lingsamuel"
+    image_url: "https://github.com/lingsamuel.png"
+keywords: 
+- Apache APISIX
+- Amesh
+- Service Mesh
+- 服务网格
+- Kubernetes
+description: Amesh 是 Apache APISIX 的服务网格库。它适配了 xDS 协议，可以从诸如 Istio 的控制平面中接收数据，并生成 APISIX 所需的数据结构，使得 APISIX 能够在服务网格领域作为数据面发挥作用。
+tags: [Ecosystem]
+---
+
+> 在云原生快速发展的前提下，服务网格领域也开始逐渐火热。目前阶段，大家所熟知的服务网格解决方案很多，每种产品又各有其优势。因此在面对不同的行业或者业务背景时，每个人的选型想法都各不相同。
+
+<!--truncate-->
+
+> 作者 [lingsamuel](https://github.com/lingsamuel)，API7.ai 云原生技术专家，Apache APISIX Committer。
+
+Apache APISIX 是一个动态、实时、高性能的云原生 API 网关，提供负载均衡、动态上游、灰度发布、服务熔断、身份认证、可观测性等丰富的流量管理功能。基于 APISIX 的扩展道路上，除了 APISIX Ingress 在云原生领域被各大厂商开始关注外，基于 APISIX 的服务网格方案也在积极迭代中。
+
+## 基于 APISIX 的服务网格方案
+
+[Amesh](https://github.com/api7/amesh) 是 [Apache APISIX](https://github.com/apache/apisix) 的服务网格库。它适配了 xDS 协议，可以从诸如 Istio 的控制平面中接收数据，并生成 APISIX 所需的数据结构，使得 APISIX 能够在服务网格领域作为数据面发挥作用。
+
+依靠 Amesh，APISIX 可以工作在服务网格模式下，不使用传统的 etcd 作为数据中心，而是使用 shdict 与 Amesh 库直接进行数据交换，避免了额外的性能损耗，使得大规模部署成为可能。
+
+通过使用 Amesh，可以在服务网格领域获得 APISIX 具备的高性能、丰富的流量管理功能、易扩展性等多种优势。
+
+### Amesh 的架构
+
+Amesh 通过适配 xDS 协议，可以让 APISIX 替代 Istio 所使用的 envoy 组件来接管集群流量。在实际使用中，APISIX 将作为 Pod 的 Sidecar 接管网格内的所有流量。目前 Amesh 的架构如下图所示：
+
+![Amesh 架构](https://static.apiseven.com/uploads/2023/01/10/VMKr49uF_image%20%2830%29.png)
```

Comment:
Get rid of the red line under "xDS"

<img width="1114" alt="image" src="https://user-images.githubusercontent.com/36651058/212676255-51bddfd0-330e-42ad-9e90-e944bb456a48.png">

Reply:
This wasn't fixed. This picture and the next picture share the same problem: a red line under 'xDS' should be removed.
 
![WeChat Screenshot_20230118094733](https://user-images.githubusercontent.com/36651058/213061208-20b314f8-cbd5-4368-a566-aab5b46eccf7.png)